Allow passing web_service_name option to Rack through Rails tracer (#2716)
DataDog/dd-trace-rb: docs/GettingStarted.md

| --- | ----------- | ------- |
| `distributed_tracing` | Enables [distributed tracing](#distributed-tracing) so that this service trace is connected with a trace of another service if tracing headers are received | `true` |
| `request_queuing` | Track HTTP request time spent in the queue of the frontend server. See [HTTP request queuing](#http-request-queuing) for setup details. | `false` |

> | `web_service_name` | Service name for frontend server request queuing spans. (e.g. `'nginx'`) | `'web-server'` |

Reviewer: ```suggestion
| `web_service_name` | Service name for frontend server request queuing spans. (such as `'nginx'`) | `'web-server'` |
```
